.github/workflows/test.yml

```diff
@@ -75,6 +78,13 @@ jobs:
           count=$(git status sample --porcelain | wc -l)
           echo "count=$count" >> $GITHUB_OUTPUT
 
+      - name: Copy the ArkanaKeys back to Sample app
```

Comment:
What do you think about `Move the ArkanaKeys to Sample app`, it likes we are moving to, not copying back 🤔

Reply:
`Restore the previous ArkanaKeys's keys` 🙏